website/source/docs/guides/common-errors.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: "docs"
+page_title: "Common Error Messages"
+sidebar_current: "docs-guides-common-errors"
+description: |-
+  Error messages you might see while installing and running Consul, and 
+  how to fix them.
+---
+
+# Common Error Messages
+
+When installing and running Consul, there are some common messages you might see. Usually they indicate an issue in your network or in your server's configuration. Some of the more common errors and their solutions are listed below.
+
+If you are getting an error message you don't see listed on this page, please consider following our general [Troubleshooting Guide][troubleshooting].
+
+## Configuration file errors
+
+```
+Multiple private IPv4 addresses found. Please configure one with 'bind' and/or 'advertise'.
+```
+Your server has multiple active network interfaces. Consul needs to know which interface to use for local LAN communications. Add the [`bind`][bind] option to your configuration. 
+
+-> **Tip**: If your server does not have a static IP address, you can use a [go-sockaddr template][go-sockaddr] as the argument to the `bind` option, e.g. `"bind_addr": "{{GetInterfaceIP \"eth0\"}}"`.
+
+---
```

Comment:
Visually, this doc has a lot of horizontal lines in it already because there are so many code blocks. You might want to change all of these horizontal lines into subheadings to break the doc up a bit more and make it more visually readable/navigable.